rules/windows/process_creation/proc_creation_win_pdqdeploy_runner_susp_children.yml

```diff
@@ -0,0 +1,36 @@
+title: Suspicious Execution Of PDQDeployRunner
+id: 12b8e9f5-96b2-41e1-9a42-8c6779a5c184
+status: experimental
+description: Detects suspicious execution of "PDQDeployRunner" which is part of the PDQDeploy service stack that is responsible for executing commands and packages on a remote machines
+references:
+    - https://twitter.com/malmoeb/status/1550483085472432128
+tags:
+    - attack.execution
+author: Nasreddine Bencherchali
+date: 2022/07/22
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_parent:
+        ParentImage|contains: 'PDQDeployRunner-'
+    selection_susp:
+        # Improve this section by adding other suspicious processes, commandlines or paths
+        - Image|endswith:
+            # If you use any of the following processes legitimately comment them out
+            - '\wscript.exe'
+            - '\cscript.exe'
+            - '\rundll32.exe'
+            - '\mshta.exe'
+            - '\csc.exe'
+            - '\powershell.exe'
```

Comment:
Do you think adding powershell.exe was a good idea? I'm unsure why you left it out.

Reply:
I left it out because I thought that in general the most common way to run admin tasks from PDQ was to use CMD or Powershell (Admin scripts). That's why I added Commnadline so for people who use PowerShell legitimately they could remove it from the image but add suspicious PowerShell CLI such as "IEX" or "DownloadString"...etc.

Reply:
That's what I expected. Yes, I can follow that thought.